runtime/Makefile.am

```diff
@@ -26,46 +26,57 @@ $(symlinks):
 SUPPORT_FILES = $(symlinks) mono-wrapper etc/mono/config
 
 build_profiles = 
+build_platforms = 
 
 if INSTALL_4_x
 build_profiles += binary_reference_assemblies net_4_x xbuild_12 xbuild_14
+build_platforms += macos win32 linux
```

Comment:
Why do you need this, seems like just duplicate work. We also need to do `make` in mono/mcs and it needs to work just like today